FlashSpace/Settings/GeneralSettingsView.swift

```diff
@@ -8,13 +8,25 @@
 import SwiftUI
 
 struct GeneralSettingsView: View {
+    @StateObject var settings = AppDependencies.shared.settingsRepository
+
     @State var isAutostartEnabled = false
 
     var body: some View {
         Form {
             Section {
                 Toggle("Launch at startup", isOn: $isAutostartEnabled)
             }
+
+            Section(
+                header: Text("Run script on bootstrap:"),
+                footer: Text(
+                    "Requires a restart to take effect."
+                )
+                .foregroundStyle(.secondary)
+            ) {
+                TextField("", text: $settings.runScriptOnBootstrap)
+            }
```

Comment:
Please move this section to `IntegrationsSettingsView.swift` and place it before the other one.